apps/console/src/layouts/app-layout.tsx

```diff
@@ -61,6 +61,9 @@ const AppLayout: FunctionComponent<Record<string, unknown>> = (): ReactElement =
         <>
             <ErrorBoundary
                 onChunkLoadError={ AppUtils.onChunkLoadError }
+                handleError={ (_error: Error, _errorInfo: React.ErrorInfo) => {
+                    sessionStorage.setItem("auth_callback_url_console", window["AppUtils"].getConfig().routes.home);
```

Comment:
Can we get the `routes.home` from the store?

Reply:
Fixed with 8e4c625